MekHQ/src/mekhq/campaign/ResolveScenarioTracker.java

```diff
@@ -1299,25 +1301,38 @@ private void initUnitsAndPilotsWithoutBattle() {
         }
     }
 
-    public ArrayList<TestUnit> getAlliedUnits() {
+    public List<TestUnit> getAlliedUnits() {
         return alliedUnits;
     }
 
-    public ArrayList<TestUnit> getPotentialSalvage() {
+    public List<TestUnit> getPotentialSalvage() {
         return potentialSalvage;
     }
 
-    public ArrayList<TestUnit> getActualSalvage() {
+    public List<TestUnit> getActualSalvage() {
         return actualSalvage;
     }
 
+    public List<TestUnit> getRansomedSalvage() {
+        return ransomedSalvage;
+    }
+
+    public List<TestUnit> getLeftoverSalvage() {
+        return leftoverSalvage;
+    }
+
     public void salvageUnit(int i) {
-        TestUnit salvageUnit = potentialSalvage.get(i);
-        actualSalvage.add(salvageUnit);
+        TestUnit salvageUnit = getPotentialSalvage().get(i);
+        getActualSalvage().add(salvageUnit);
     }
 
-    public void dontSalvageUnit(int i) {
-        leftoverSalvage.add(potentialSalvage.get(i));
+    public void ransomUnit(int i) {
+        TestUnit ransomUnit = potentialSalvage.get(i);
```

Comment:
These are public and should probably include a check on `i`.

Reply:
They are used by the ResolveScenarioWizardDialog, so they need to be. I've added the check on the i value to be safe, but it is not necessary based on how they are used at this time.